whatsnext/models.py

```diff
@@ -47,8 +30,32 @@ def _get_full_name(self):
     
     full_name = property(_get_full_name)
 
+class Task(models.Model):
+    id = models.IntegerField(primary_key=True)
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    task_description = models.CharField(max_length=1000)
+    priority = models.IntegerField()
+    # Later probably want to have a separate categories table to allow
+    # for users to create their own categories.
+    # But for now just have dumb integer field to keep the dream alive.
+    category = models.IntegerField()
+    created = models.DateTimeField(default=datetime.datetime.now)
+    status = models.IntegerField(choices=STATUSES, default=1)
+
+    def __str__(self):              
+        return self.task_description
+
+    def status_status(self):
+        "Returns the task's status."        
+        import datetime        
+        if self.status == 0:
+            return "Open"
```

Comment:
I don't think we should hard code these in two places. If you want STATUSES, then make it a dictionary instead of an array of tuples, and make the integers the keys and just return the values here.